ebpf/port_test.go

```diff
@@ -0,0 +1,54 @@
+package ebpf
+
+import (
+	"github.com/stretchr/testify/require"
+	"net"
+	"net/url"
+	"strconv"
+	"testing"
+)
+
+func TestReadInitialState(t *testing.T) {
+	l, err := net.Listen("tcp", ":0")
+	require.NoError(t, err)
+	defer func() { _ = l.Close() }()
```

Comment:
`defer l.Close()` is not working here ? Or is to avoid errors about not checking for the returned error ?

Reply:
> Or is to avoid errors about not checking for the returned error ?

Yes, this one.